src/PyCall.jl

```diff
@@ -74,6 +79,18 @@ mutable struct PyObject
     end
 end
 
+PyPtr(o::PyObject) = getfield(o, :o)
+
+"""
+    ≛(x, y)
+
+`PyPtr` based comparison of `x` and `y`, which can be of type `PyObject` or `PyPtr`. 
+"""
+≛(o::PyObject, p::PyPtr) = PyPtr(o) == p
+≛(p::PyPtr, o::PyObject) = o ≛ p
+≛(o1::PyObject, o2::PyObject) = PyPtr(o1) == PyPtr(o2)
+≛(p1::PyPtr, p2::PyPtr) = p1 == p2
```

Comment:
You could just have one method:
```jl
≛(o1::Union{PyObject,PyPtr}, o2::Union{PyObject,PyPtr}) = PyPtr(o1) == PyPtr(o2)
```
since `PyPtr(p::PyPtr) = p` should be defined automatically.